### PR TITLE
made compatible with ubuntu 20.04

### PR DESCRIPTION
**Pull request by nazmulshuvo03**
1) install_prerequisites.sh file required libgfortra3
2) demo scripts was installing libpng12-dev,
which are not compitable with ubuntu later 16 versions.

### CODE REVIEW

File: model-optimizer/install_prerequisites/install_prerequisites.sh [hunk 1]
```diff
@@ -72,7 +72,7 @@ if [[ $DISTRO == "centos" ]]; then
     fi
 elif [[ $DISTRO == "ubuntu" ]]; then
     sudo -E apt update
-    sudo -E apt -y install python3-pip python3-venv libgfortran3
+    sudo -E apt -y install python3-pip python3-venv libgfortran5
```
Comment (alalek) [inline model-optimizer/install_prerequisites/install_prerequisites.sh:75]: > libgfortran*

@artanokhov @asuhov Do we really need to expose this dependency directly? (which OV component requires that?)

File: scripts/demo/demo_benchmark_app.sh [hunk 2]
```diff
@@ -122,6 +122,8 @@ elif [[ $DISTRO == "ubuntu" ]]; then
     system_ver=`cat /etc/lsb-release | grep -i "DISTRIB_RELEASE" | cut -d "=" -f2`
     if [ $system_ver = "18.04" ]; then
         sudo -E apt-get install -y libpng-dev
+    elif [ $system_ver = "20.04" ]; then
+        sudo -E apt-get install -y libpng-dev
     else
         sudo -E apt-get install -y libpng12-dev
```
Comment (alalek) [inline scripts/demo/demo_benchmark_app.sh:128]: Lets refactor this section:
```
if [ $system_ver = "16.04" ]; then
    sudo -E apt-get install -y libpng12-dev
else
    # Ubuntu 18.04+
    sudo -E apt-get install -y libpng-dev
fi
```

Comment (nazmulshuvo03) [reply scripts/demo/demo_benchmark_app.sh:128]: @alalek this will install libpng-dev for ubuntu 14 and other older versions. and we don't want that.

Comment (alalek) [reply scripts/demo/demo_benchmark_app.sh:128]: I know, but these EOL versions are not supported here anymore (at least on master branch).

Comment (nazmulshuvo03) [reply scripts/demo/demo_benchmark_app.sh:128]: @alalek created another PR ( #1827 ) with the fixes you suggested

Comment (nazmulshuvo03) [reply scripts/demo/demo_benchmark_app.sh:124]: @ilya-lavrenov I have closed that PR and updated this one

**Review by postrational — APPROVED** [review]

**Comment by alalek** [comment]
> scripts/demo/18.04 

This unwanted change is still here.

**Comment by nazmulshuvo03** [comment]
> > scripts/demo/18.04
> 
> This unwanted change is still here.

@alalek sorry for that. I couldn't figure out the the empty file earlier. Now I have removed it. Please check again.

File: model-optimizer/install_prerequisites/install_prerequisites.sh (hunk 1 above)
Comment (asuhov) [reply model-optimizer/install_prerequisites/install_prerequisites.sh:75]: libgfortran3 package was required to install MXNet python module for model optimizer

**Review by asuhov — APPROVED** [review]

**Pull request merged**